res/values/strings.xml

```diff
@@ -712,4 +756,27 @@
 	<string name="ConversationFragment_quoted_message_not_found">Original message not found</string>
 	<string name="reply_privately">Reply privately</string>
 
+    <!-- Emoji picker and categories -->
+    <string name="emoji_search_results">Search results</string>
+    <string name="emoji_not_found">No emoji found</string>
+    <string name="emoji_recent">Recently Used</string>
+    <string name="emoji_people">People & Body</string>
```

Comment:
the ampersand looks like it wants to be escaped, does xml use the html notation for this, too?

Reply:
yes (`&amp;`)